docs/setup-cluster/k8s/resource-caps.rst

```diff
@@ -0,0 +1,135 @@
+.. _k8s-resource-caps:
+
+#####################################
+ Manage Workspace Namespace Bindings
```

Comment:
Should we call it workspace - namespace bindings? Or Workspace <> Namespace bindings or something?

Reply:
yes it makes sense to add a hyphen